sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -361,6 +360,11 @@ namespace Azure { namespace Storage { namespace Blobs {
     throw std::runtime_error("cannot convert " + blob_lease_status + " to BlobLeaseStatus");
   }
 
+  struct BlobPrefix
```

Comment:
Why is this a separate struct?

Reply:
aligned with other sdk

Reply:
I don't think it makes sense, I don't want to always assume that other SDKs are carefully thought about.

Reply:
```
    <BlobPrefix>  
      <Name>blob-prefix</Name>  
    </BlobPrefix>  
```
here's a snippet of the HTTP response. `Name` is just a property of `BlobPrefix`, although currently the only one. We cannot rule out the possibility of more properties being added in the future.

So I think it makes sense to define a struct that maps to this xml node.